index.html

```diff
@@ -1,27 +1,101 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <title>Fork Me! FCC: Test Suite Template</title>
-    <link rel="stylesheet" href="css/style.css" />
-  </head>
 
-  <body>
-    <h1>Hello world</h1>
-    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+<head>
+  <meta charset="UTF-8" />
+  <title>Fork Me! FCC: Test Suite Template</title>
+  <link rel="stylesheet" href="css/style.css" />
+</head>
 
-    <!--
+<body>
+  <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
 
-    Hello Camper!
+  <header id="header">
+    <img src="https://cdn.freecodecamp.org/testable-projects-fcc/images/product-landing-page-logo.png"
+      id="header-img" />
 
-    For now, the test suite only works in Chrome!
-    Please read the README below in the JS Editor before beginning.
-    Feel free to delete this message once you have read it.
-    Good luck and Happy Coding!
+    <nav id="nav-bar">
+      <ul>
+        <li><a href="#Features" class="nav-link">Features</a></li>
+        <li><a href="#video" class="nav-link">How it Works</a></li>
+        <li><a href="#Pricing" class="nav-link">Pricing</a></li>
+      </ul>
+    </nav>
+  </header>
+  <main>
+    <form id="form" action="https://www.freecodecamp.com/email-submit">
+      <h1>Handcrafted, home-made masterpieces</h1>
```

Comment:
h1 elements are used to define the main heading of a webpage for semantic reasons, kindly use a different tag like h2 or others.